packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -237,7 +237,11 @@ class CupertinoPageRoute<T> extends PageRoute<T> {
 
   @override
   Widget buildPage(BuildContext context, Animation<double> animation, Animation<double> secondaryAnimation) {
-    final Widget result = builder(context);
+    final Widget result = new Semantics(
+      route: true,
+      explicitChildNodes: true,
+      child: builder(context)
+    );
```

Comment:
same comments as for material page route.

Reply:
removed logic for semantics name in pages